src/web/client/common/worker/webworker.js

```diff
@@ -146,10 +162,25 @@ async function loadContainer(config, swpId, entityInfo) {
                     containerId: swpId,
                     entityId: userEntityIdArray,
                 });
+
+                await self.postMessage({
+                    type: "telemetry",
+                    eventName: "client-data success",
```

Comment:
is this event name aligning with current naming convention? Refer current event constants.

Reply:
addressed